.github/workflows/release.yaml

```diff
@@ -0,0 +1,108 @@
+# This workflow is based on the workflow by automl/amltk under the BSD-3-Clause license:
+#   Copyright 2023 AutoML-Freiburg-Hannover-Tübingen
+#   https://github.com/automl/amltk/blob/c3eb4ac29c50bdbdf3c1e19a3c99a8a1e0794fba/LICENSE
+
+name: release
+on:
+  workflow_dispatch:
+    inputs:
+      increment:
+        description: "Version segment to increment"
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+        default: patch
+
+permissions:
+  contents: write
+jobs:
+  test-code:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+          cache: pip
+      - run: python -m pip install ".[dev]"
+      - run: python -m pytest tests
+  bump:
+    needs: [test-code]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-tags: 1  # Essential to later commitizen
+          fetch-depth: 0  # Recommended by the action
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - run: git tag  # Debug statement
```

Comment:
Do we really need to keep this? It's just for debugging, right?

Reply:
Correct. It was in the original workflow and I opted to leave it in since it is very useful if something does go wrong, and it's basically free (close to 0 second runtime).